scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   // below the smallest value in the build output; this avoids the
   // need for frequent changes as bundle sizes fluctuate.
   MIN_SIZE: 673,
-  MAX_SIZE: 851,
+  MAX_SIZE: 855,
```

Comment:
the bundlesize has tipped over into 852, I've put it up to 855, in keeping with expectation to be around 5 kb the actual bundle size